src/modules/helpchan.ts

```diff
@@ -569,4 +571,83 @@ export class HelpChanModule extends Module {
 		await this.ensureAskChannels(msg.guild);
 		await msg.channel.send(':ok_hand:');
 	}
+
+	getDormantChannels(guild: Guild) {
+		return guild.channels.cache
+			.filter(channel => channel.parentID == categories.dormant)
+			.filter(channel => channel.name.startsWith(this.CHANNEL_PREFIX));
+	}
+
+	getOutdatedChannels(guild: Guild) {
+		return this.getOngoingChannels()
+			.concat(
+				this.getAvailableChannels(guild).array() as TextChannel[],
+				this.getDormantChannels(guild).array() as TextChannel[],
+			)
+			.filter(it => it.topic !== helpChannelTopic);
+	}
+
+	updateHelpChannelTopicTimeout: NodeJS.Timeout | undefined = undefined;
+
+	@command({
+		inhibitors: [CommonInhibitors.hasGuildPermission('MANAGE_MESSAGES')],
+	})
+	async updateHelpChannelTopics(msg: Message) {
+		const guild = msg.guild;
+		if (!guild || msg.channel.type !== 'text') return;
```

Comment:
It could be a `NewsChannel` or `DMChannel`… but I think the actual reason is that I was anticipating that I'd need to pass it to one of the functions that expects a `TextChannel`, but that never happened. I'll remove it.